Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -3791,6 +3792,83 @@ def com_google_fonts_check_name_family_and_style_max_length(ttFont):
     yield PASS, "All name entries are good."
 
 
+@check(
+    id='com.google.fonts/check/has_unacceptable_control_chars',
+    conditions=['are_ttf']
+)
+def com_google_fonts_check_has_unacceptable_control_chars(ttFonts):
+  """Does font file include unacceptable control character glyphs?
+
+  Args:
+      ttFonts (list): list of one or more fontTools.ttLib.TTFont objects
+  """
+  # list of unacceptable control character glyph names
+  # definition includes the entire control character Unicode block except:
+  #    - .null (U+0000)
+  #    - CR (U+000D)
+  unacceptable_cc_list = [
+      "uni0001",
+      "uni0002",
+      "uni0003",
+      "uni0004",
+      "uni0005",
+      "uni0006",
+      "uni0007",
+      "uni0008",
+      "uni0009",
+      "uni000A",
+      "uni000B",
+      "uni000C",
+      "uni000E",
+      "uni000F",
+      "uni0010",
+      "uni0011",
+      "uni0012",
+      "uni0013",
+      "uni0014",
+      "uni0015",
+      "uni0016",
+      "uni0017",
+      "uni0018",
+      "uni0019",
+      "uni001A",
+      "uni001B",
+      "uni001C",
+      "uni001D",
+      "uni001E",
+      "uni001F"
+  ]
+
+  # a dict with key:value of font path that failed check : list of unacceptable glyph names
+  failed_font_dict = {}
+
+  for tt in ttFonts:
```

Comment:
This is just a matter of taste, but we have used `for ttFont in ttFonts:` throughout the rest of the codebase so could be nice to do the same here.

Reply:
refactored!